### DISCUSSION TITLE
Phalcon 5 PHQL/QueryBuilder/Model will work in the same way?

### DISCUSSION THREAD

**Discussion by renatogabrielbr** [phalcon/phalcon#99] [General]
HelloI want to know if phalcon 5 will have  PHQL, QueryBuilder and Model work in the same  way as phalcon 4.1 or we will need to refactor the code?

Thanks.

**Comment by Jeckerson** [comment]
Yes, v5 will be almost identical as v4.1. As v5 is transitional version from C ext to PHP native.

**Reply by renatogabrielbr** [reply]
Nice, thank you.
